src/nplinker/pairedomics/downloader.py

```diff
@@ -25,6 +25,7 @@
 class PODPDownloader():
 
     def __init__(self, platform_id, force_download=False, local_cache=None):
+        # platform_id must be gnps_massive_id, it should be validated
```

Comment:
Are we validating it in this class? If yes, I'd mention here the method in which it's validated

Reply:
this comment is removed later, the platform id can be any type of mass spectrum database id, including massive id.